cloud_config/.env.catalog.cloud

```diff
@@ -0,0 +1,9 @@
+APP_URL=https://<app_dns_name>
+CATALOG_URL=https://<catalog_dns_name>/api
+CATALOG_DOCBASEURL=https://www.matatika.com/docs
+CATALOG_WEBSITEBASEURL=https://www.matatika.com
+CATALOG_ALLOWED_ORIGINS=https://<app_dns_name>
+CATALOG_AUTH_IDPS_LOCAL_ISSUER_URI=https://<catalog_dns_name>/
+CATALOG_AUTH_IDPS_LOCAL_AUDIENCE=https://<catalog_dns_name>/api
+CATALOG_AUTH_IDPS_PRIMARY_AUDIENCE=https://<catalog_dns_name>/api
```

Comment:
Probably redundant when local audience is supplied.  These all go into the same list of identify providers